cms/jinja2/templates/base.html

```diff
@@ -56,10 +57,54 @@
     }
 }
 %}
+
 {# fmt:on #}
 {% block head %}
     <link rel="stylesheet" href="{{ static('css/main.css') }}">
-{% endblock %}
+
+  <!-- Google Tag Manager -->
```

Comment:
question: do we need the HTML comments?

Reply:
I've removed these as now with the conditional it should be clear from the context what that is related to. 👍